src/types/governance_converters.ts

```diff
@@ -104,8 +104,10 @@ export interface Followees {
   topic: Topic;
   followees: Array<NeuronId>;
 }
+
 export interface GovernanceError {
   errorMessage: string;
+  // TODO: What does this number mean?
   errorType: number;
```

Comment:
@hpeebles or @ielashi do you know what this `errorType` number represents? It comes from the `GovernanceError#error_type` from the canister.

If you don't have it in top of your mind, I can check the Governance Canister code.

Reply:
The description finds place in the governance canister (see for example not_found -> https://github.com/dfinity/ic/blob/48c3904e38f6708bcf89f498ada340cd8c372f57/rs/nns/governance/tests/governance.rs#L8567)

I did not went deeper but I guess somewhere `ErrorType` should be defined and list all the error types. Not sure it is in the IC though, like I said, did not went deeper.

Reply:
You can see the full list of error types here - https://github.com/dfinity/ic/blob/48c3904e38f6708bcf89f498ada340cd8c372f57/rs/nns/governance/proto/ic_nns_governance/pb/v1/governance.proto#L911

Reply:
thanks, I didn't know the governance canister was inside nns.